.github/workflows/release.yml

```diff
@@ -0,0 +1,167 @@
+          path: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GOVERSION }}
+
+      - name: Build armadactl release artifacts
+        run: |
+          TAG=${GITHUB_REF##*/}
+          make build-armadactl-release RELEASE_VERSION=${TAG}
+        working-directory: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+
+      - name: Archive production artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: ${{ env.GOPATH }}/src/github.com/G-Research/armada/dist/*
+
+      #### Publish release
+      - name: Upload artifacts to Github release
+        uses: ncipollo/release-action@v1.8.6
+        with:
+          artifacts: "${{ env.GOPATH }}/src/github.com/G-Research/armada/dist/*"
+          token: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ github.event.release.tag_name }} 
+
+      - name: Push Release Image
+        run: |
+          if [ -z "${DOCKERHUB_USER}" ]
+            then
+              echo "Do not push image inside fork."
+              exit 0
+          fi
+
+          TAG=${GITHUB_SHA}
+          RELEASE_TAG=${GITHUB_REF##*/}
+
+          echo ${DOCKERHUB_PASS} | docker login -u ${DOCKERHUB_USER} --password-stdin
+
+
+          docker pull gresearchdev/armada-server-dev:${TAG}
+          docker tag gresearchdev/armada-server-dev:${TAG} gresearchdev/armada-server:${RELEASE_TAG}
+          docker push gresearchdev/armada-server:${RELEASE_TAG}
+
+          docker pull gresearchdev/armada-executor-dev:${TAG}
+          docker tag gresearchdev/armada-executor-dev:${TAG} gresearchdev/armada-executor:${RELEASE_TAG}
+          docker push gresearchdev/armada-executor:${RELEASE_TAG}
+
+          docker pull gresearchdev/armada-armadactl-dev:${TAG}
+          docker tag gresearchdev/armada-armadactl-dev:${TAG} gresearchdev/armada-armadactl:${RELEASE_TAG}
+          docker push gresearchdev/armada-armadactl:${RELEASE_TAG}
+
+          docker pull gresearchdev/armada-lookout-dev:${TAG}
+          docker tag gresearchdev/armada-lookout-dev:${TAG} gresearchdev/armada-lookout:${RELEASE_TAG}
+          docker push gresearchdev/armada-lookout:${RELEASE_TAG}
+
+          docker pull gresearchdev/armada-binoculars-dev:${TAG}
+          docker tag gresearchdev/armada-binoculars-dev:${TAG} gresearchdev/armada-binoculars:${RELEASE_TAG}
+          docker push gresearchdev/armada-binoculars:${RELEASE_TAG}
+
+          rm -v /home/runner/.docker/config.json
+        working-directory: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+
+  update-charts:
+    needs: [release]
+    name: Publish Armada charts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Armada
+        uses: actions/checkout@v2
+        with:
+          path: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+
+      - name: Clone chart repo, update version and image tag to match current $RELEASE_TAG
```

Comment:
@ljubon can we split this step into several steps? for example:
- setup ssh
- checkout charts to path similar to [step above](https://github.com/ljubon/armada/pull/83/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R94)
- execute other part of script with helm

What do you think?

Reply:
i don't see the reason why we should do this, as in a few days/weeks we will replace it with a new chart process (and chart repo will change/update its structure sooner)

Reply:
Still valid comment, let's see how to proceed with this

Reply:
You are right. Because of that, i would leave this PR open until then, come back to it, and see how to change it. Is this ok?

Reply:
alright :) will keep this comment not-resolved